representable/settings/prod.py

```diff
@@ -10,14 +10,16 @@
 
 DEBUG = False
 
-ACCOUNT_EMAIL_VERIFICATION = "mandatory"
-
-ACCOUNT_EMAIL_CONFIRMATION_EXPIRE_DAYS = 7
+# Production Email Settings
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
```

Comment:
is the EMAIL_BACKEND line necessary in prod? I don't see it in the version that's currently live

Reply:
It's not but I think it's useful to be explicit here (it's the default). I often find myself wondering whether by removing it I break the system.